jujugui/static/gui/src/app/components/profile/model-list/test-model-list.js

```diff
@@ -699,6 +699,149 @@ describe('Profile Model List', function() {
     expect(output).toEqualJSX(expected);
   });
 
+  it('does not show the trash icon for controller models', () => {
```

Comment:
Should we implement this kind of checks with enzyme now?

Reply:
yes but I didn't want to do the enzyme conversion in this branch. So I figured we'd convert it all in one swoop.